src/main/common/draft/draftService.ts

```diff
@@ -0,0 +1,32 @@
+import DraftStoreClient from 'common/draft/draftStoreClient'
+
+import { Draft } from 'app/models/draft'
+import { DraftStoreClientFactory } from 'common/draft/draftStoreClientFactory'
+import { CoreOptions, RequestAPI } from 'request'
+import { RequestPromise } from 'request-promise-native'
+import { ServiceAuthTokenFactory } from 'common/security/serviceTokenFactory'
+
+export class DraftService {
+  constructor (public draftStoreUri: string, public request: RequestAPI<RequestPromise, CoreOptions, CoreOptions>, public serviceAuthTokenFactory: ServiceAuthTokenFactory) {
```

Comment:
Are there no checks on max line length?

Can you set up CI before this gets merged (run tests, linter etc)?